bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/discovery/HueBridgeNupnpDiscovery.java

```diff
@@ -87,37 +75,25 @@ private void discoverHueBridges() {
         for (BridgeJsonParameters bridge : getBridgeList()) {
             if (isReachableAndValidHueBridge(bridge)) {
                 String host = bridge.getInternalIpAddress();
-                String serialNumber = bridge.getId().substring(0, 6) + bridge.getId().substring(10);
-                serialNumber = serialNumber.toLowerCase();
+                String serialNumber = bridge.getId().toLowerCase();
```

Comment:
You are changing the way to get the serial number. Do you mean this was not working well ?

Reply:
The removed UPnP discovery used a shortened `bridgeid` without the bridge indicator "fffe" in it. This has changed in mDNS discovery and thus we do not need this logic anymore to align the two bridge discovery services.